app/controllers/rents_controller.rb

```diff
@@ -0,0 +1,22 @@
+class RentsController < ApplicationController
+  respond_to :json
+  before_action :authenticate_user!
+
+  def index
+    rent = current_user.rents
```

Comment:
Change this to `rents` since it's an array.

Reply:
Done